src/applications/check-in/day-of/pages/LoadingPage/index.jsx

```diff
@@ -1,95 +1,34 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
-import { useDispatch, useSelector, batch } from 'react-redux';
+import React, { useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
 import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
+import { useGetCheckInData } from '../../../hooks/useGetCheckInData';
 import { useFormRouting } from '../../../hooks/useFormRouting';
-import {
-  receivedDemographicsData,
-  receivedMultipleAppointmentDetails,
-  triggerRefresh,
-  updateFormAction,
-} from '../../../actions/day-of';
-import { useSessionStorage } from '../../../hooks/useSessionStorage';
-import {
-  makeSelectCurrentContext,
-  makeSelectVeteranData,
-} from '../../../selectors';
-import { api } from '../../../api';
+import { makeSelectVeteranData } from '../../../selectors';
+
+const { isEmpty } = require('lodash');
```

Comment:
I like the elegance of this over the pure JS alternatives but I'm definitely open to argument that we shouldn't bring in lodash just for this :)

Reply:
you could also argue that we should use lodash more, since it is already there and loaded. I like this and we can always factor it out if platform makes a way for us to split out unused libraries and we aren't using lodash enough to make it worth while.